README.md

```diff
@@ -23,17 +23,9 @@ Djinni **requires Java** to be able to execute!
 
 ### Linux, macOS
 
-#### Download & Install Manually
+#### Manually
 
-Download the [lastest released binary](https://github.com/cross-language-cpp/djinni-generator/releases/latest/download/djinni), make it executable and put the containing folder on your PATH:
-
-```bash
-curl -LO https://github.com/cross-language-cpp/djinni-generator/releases/latest/download/djinni
-chmod u+x djinni
-export PATH=$(pwd):$PATH
-```
-
-Make sure to [put it on your PATH permanently](https://stackabuse.com/how-to-permanently-set-path-in-linux/) once you fell in love with djinni! :blush:
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/cross-language-cpp/djinni-generator?label=Download&logo=linux&logoColor=%23fff&style=for-the-badge)](https://github.com/cross-language-cpp/djinni-generator/releases/latest/download/djinni)
```

Comment:
Somehow the install instructions seemed bloated to me. Then I came up with this idea. It utilizes shields.io badges as download buttons and looks awesome in my opinion!!!

<img width="908" src="https://user-images.githubusercontent.com/21294002/113492543-5849a400-94d8-11eb-8f58-c73a3a554199.png">